src/components/progress-report/workflow/progress-report-workflow.edgedb.repository.ts

```diff
@@ -0,0 +1,193 @@
+import { Injectable } from '@nestjs/common';
+import {
+  ID,
+  NotFoundException,
+  PublicOf,
+  RichTextDocument,
+  Session,
+  UnsecuredDto,
+} from '~/common';
+import { e, RepoFor } from '~/core/edgedb';
+import { ProgressReportStatus as Status } from '../dto';
+import { ProgressReportWorkflowEvent } from './dto/workflow-event.dto';
+import { ProgressReportWorkflowRepository } from './progress-report-workflow.repository';
+import { InternalTransition } from './transitions';
+
+@Injectable()
+export class ProgressReportWorkflowEdgeDBRepository
+  extends RepoFor(ProgressReportWorkflowEvent, {
+    hydrate: (event) => ({
+      ...event['*'],
+      report: true,
+      who: true,
+      transition: event.transitionId,
+    }),
+  }).customize((cls, { defaults }) => {
+    return class extends cls {
+      static omit = [defaults.create, defaults.update, defaults.delete];
+      async list(
```

Comment:
@CarsonF - Andre's reasoning for customizing this is because the signature in the common EdgeDB repo uses Pagination, but the one in the Neo4j repo does not.  Is customizing the best approach here?  Should we consider altering the Neo4j list instead?

Reply:
The list isn't paginated in GQL, so we couldn't just change the neo4j query.
https://github.com/SeedCompany/cord-api-v3/blob/develop/src/components/progress-report/workflow/resolvers/progress-report-workflow-events.resolver.ts#L11-L17

Customizing the method here is the right approach.